gvisor/README.md

```diff
@@ -0,0 +1,53 @@
+# gVisor extension
+
+## Usage
+
+Enable the extension in the machine configuration before installing Talos:
+
+```yaml
+machine:
+  install:
+    extensions:
+      - image: ghcr.io/talos-systems/gvisor:<VERSION>
+```
+
+gVisor requires unprivileged user namespace creation, so Talos default setting
+should be overridden:
+
+```yaml
+machine:
+  sysctls:
+    user.max_user_namespaces: "11255"
+```
+
+> Warning! This disables [KSPP best practices](https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings#sysctls) setting.
+
+## Testing
+
+Apply the following manifest to run nginx pod via gVisor:
+
+```yaml
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: gvisor
+handler: runsc
```

Comment:
First time I have seen a Kubernetes resource have something other than metadata or spec at this level in the YAML. Just pointing this out in case it is a typo.

Reply:
it works :man_shrugging: that's the only thing I can say in my defense :-D